src/app/lib/utilities/variantHandler/index.js

```diff
@@ -5,7 +5,7 @@ export const servicesWithVariants = {
   zhongwen: ['simp', 'trad'],
 };
 
-export const variants = ['simp', 'trad', 'lat', 'cyr'];
+export const variants = ['simp', 'trad', 'lat', 'cyr', 'default'];
```

Comment:
I've added `default` here because of this PR: https://github.com/bbc/psammead/pull/2704

This will mean we will not get prop erros from this https://github.com/bbc/simorgh/pull/4797/files#diff-f6b3a30741ee40e9adab6047e118a34bR14
wheb variant = default